Formula/q/qt@5.rb

```diff
@@ -24,9 +28,9 @@ class QtAT5 < Formula
 
   keg_only :versioned_formula
 
-  depends_on "node"       => :build
+  depends_on "node" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.10" => :build
+  depends_on "python@3.11" => :build # NOTE: Python 3.12+ would need additional backports due to imp usage
```

Comment:
might be good to link any open tickets on `imp` usage or py3.12 support

Reply:
There are commits scattered around Chromium 118 (https://github.com/chromium/chromium/commit/f5f6e361d037c31630661186e7bd7b31d2784cb8), 120 (https://github.com/chromium/chromium/commit/9e0c89a3b5638ba2b9b107fea34a01c774aa7805), etc. However, I doubt these will get backported to `qt@5` as upstream only does security backports (thus code base still uses Python 2).

More likely to get it in `qt` 6. That is still on 112 in current release (https://github.com/qt/qtwebengine/blob/v6.6.1/CHROMIUM_VERSION) but they at least upgrade base version on minor releases.

Issue ref: https://bugs.chromium.org/p/chromium/issues/detail?id=1487454